### PR TITLE
Fixes #16022: Reload relayd when updating nodeslist.json on relays

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16022

### CODE REVIEW

File: techniques/system/distributePolicy/1.0/propagatePromises.cf [hunk 1]
```diff
@@ -117,14 +117,21 @@ bundle agent propagatePromises
         classes => if_else("nodeslist_copied", "could_not_copy_nodeslist");
 
   methods:
+
+    policy_server.!root_server.nodeslist_copied::
+      "any" usebundle => service_reload("rudder-relayd");
```
Comment (ncharles) [inline techniques/system/distributePolicy/1.0/propagatePromises.cf]: this will be done at each run: the if_else defines nodeslist_copied even if it was ok

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**